### PR TITLE
storage: Anaconda mounting improvements

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

File: pkg/storaged/btrfs/subvolume.jsx [hunk 1]
```diff
@@ -255,6 +257,13 @@ export function make_btrfs_subvolume_page(parent, volume, subvol) {
     const forced_options = [`subvol=${subvol.pathname}`];
     const mount_point_in_parent = get_mount_point_in_parent(volume, subvol);
 
+    if (client.in_anaconda_mode()) {
+        actions.push({
+            title: _("Edit mount point"),
+            action: () => edit_mount_point(block, forced_options, subvol),
```
Comment (cockpituous) [inline pkg/storaged/btrfs/subvolume.jsx:263]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19973-20240208-140838-896b0145-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/filesystem/utils.jsx [hunk 2]
```diff
@@ -212,10 +216,8 @@ export const MountPoint = ({ fstab_config, forced_options, backing_block, conten
                 <FlexItem>{ mount_point_text }</FlexItem>
                 }
                 <FlexItem>
-                    <StorageLink onClick={() => mounting_dialog(client,
-                                                                content_block || backing_block,
-                                                                "update",
-                                                                forced_options, subvol)}>
+                    <StorageLink onClick={() => edit_mount_point(content_block || backing_block,
```
Comment (cockpituous) [inline pkg/storaged/filesystem/utils.jsx:219]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19973-20240208-140838-896b0145-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by KKoukiou — APPROVED** [review]
Regarding editing of mount points - I think it's enough that the people will re-format the device in anaconda mode. 

but as you wish.

**Comment by mvollmer** [comment]
> Regarding editing of mount points - I think it's enough that the people will re-format the device in anaconda mode.

Yes, they can also go to the details page of the filesystem and click on the "edit2 button next to the mount point description.

But in my own manual testing, I always found it distracting: You format something and don't put in the mount point, and the you look at your new filesystem and it says "no mount point assigned", and you think,"yeah, let's assign a mount point", but there is no obvious way to do that.

Formatting it again and filling out the mount point this time is one way, but it feels like a bit of a punishment. Computer says "You did it wrong, do it again.", kind of.

**Pull request merged**